src/Bridge/LaravelDoctrine/Extensions/LoggableExtension.php

```diff
@@ -75,12 +76,14 @@ private function getUsernameResolver(): Closure
                 return 'testing';
             }
 
-            if ($this->auth->user() === null) {
+            $user = $this->auth->user();
+
+            if (($user instanceof EntityInterface) === false) {
```

Comment:
Would we not check for `UserInterface` here, instead?

Reply:
agreed, good find. fixed.